native/Avalonia.Native/src/OSX/AvnWindow.mm

```diff
@@ -492,7 +492,7 @@ - (void)sendEvent:(NSEvent *_Nonnull)event
                     auto point = [self translateLocalPoint:avnPoint];
                     AvnVector delta = { 0, 0 };
 
-                    parent->BaseEvents->RawMouseEvent(NonClientLeftButtonDown, static_cast<uint64>([event timestamp] * 1000), AvnInputModifiersNone, point, delta);
+                    parent->BaseEvents->RawMouseEvent(NonClientLeftButtonDown, AvnPointerDeviceType::Mouse, static_cast<uint64>([event timestamp] * 1000), AvnInputModifiersNone, point, delta, 1.0f, .0f, .0f);
```

Comment:
1.0 should be 0.5 to match the default for mouse on other platforms (also see the previous comment).